### PR TITLE
R uniqueatommat

### PR DESCRIPTION
**Pull request by MathiasHaudgaard**

### CODE REVIEW

**Comment by jdblischak** [comment]
@ocefpaf Could you please restart this CircleCI build?

**Comment by jdblischak** [comment]
xref: https://github.com/conda-forge/staged-recipes/issues/4320

**Comment by bgruening** [comment]
It triggered it.

File: recipes/r-uniqueatommat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  license: GPL (>= 2)
+  summary: 'An alternative implementation of base::duplicated.matrix(), base:anyDuplicated.matrix()
+    and base::unique.matrix() for matrices of atomic mode, avoiding the time consuming
+    collapse of the matrix into a character vector. An extra grpDuplicated() function
+    is added, which groups duplicated elements together. '
+
+  license_family: GPL3
+extra:
+   recipe-maintainers:
+    - MathiasHaudgaard
```
Comment (bgruening) [inline recipes/r-uniqueatommat/meta.yaml]: @MathiasHaudgaard please consider to add the others from the conda_r_helper as well.

**Comment by bgruening** [comment]
it seems that this package is not able to build against older R versions
`ERROR: this R is version 3.3.2, package 'uniqueAtomMat' requires R >=  3.4.0`

Can you manually skip R builds? `>= 3.4.1`

**Comment by MathiasHaudgaard** [comment]
I'm not sure how to do that.
Why is the R version at 3.3.2 instead of the latest version?

**Comment by bgruening** [comment]
Because conda-forge is currently building against 3.3.2 and 3.4.1.

**Comment by bgruening** [comment]
Just add `- r-base >=3.4.1` to your recipe.

**Comment by MathiasHaudgaard** [comment]
Ok, I'll do that. Thanks :-)

**Pull request closed without merging**